modules/README.md

```diff
@@ -75,3 +75,5 @@ $ cmake -D OPENCV_EXTRA_MODULES_PATH=<opencv_contrib>/modules -D BUILD_opencv_<r
 - **xobjdetect**: Boosted 2D Object Detection -- Uses a Waldboost cascade and local binary patterns computed as integral features for 2D object detection.
 
 - **xphoto**: Extra Computational Photography -- Additional photo processing algorithms: Color balance / Denoising / Inpainting.
+
+- **Julia**: Julia Interface -- OpenCV Julia wrapping code with some basic functionality and samples.
```

Comment:
make **julia** lower case. the word OpenCV is unnecessary here, everything in opencv_contrib is about OpenCV. i would rename it to "Julia language wrappers with samples and tests"

Reply:
Done